data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -184,12 +184,23 @@
             id=Engineer
             name= _ "Engineer"
             unrenamable=yes
-            type=Red Mage
+            type={ON_DIFFICULTY "Arch Mage" "Silver Mage" "Red Mage"}
```

Comment:
I think it's better to not vary the type based on difficulty. Some people may prefer silver, some arch. (I personally almost always use silver mages)

Reply:
reverted in 33a48a5